utils/kata-manager.sh

```diff
@@ -973,7 +1052,7 @@ handle_args()
 	local containerd_flavour="lts"
 	local kata_tarball=""
 
-	while getopts "c:dDfhk:K:lortT" opt "$@"
+	while getopts "c:dDfhk:K:lortT:N" opt "$@"
```

Comment:
This looks like it's got an extraneous colon in it? Also, let's keep the options in alphabetic order ;)

```suggestion
	while getopts "c:dDfhk:K:lNortT" opt "$@"
```